VimScriptForPythonDevelopers.MD

```diff
@@ -100,25 +104,15 @@ addition| a = 10 + 20 | let a = 10 + 20
 subtraction| a = 30 - 10 | let a = 30 - 10
 multiplication| a = 3 * 5 | let a = 3 * 5
 division| a = 22 / 7 | let a = 22 / 7
-modulus| a = 10 % 3 | let a = 10 % 3
-exponentiation| a = 2 \*\* 3 | let a = float2nr(pow(2, 3))
-floor division| a = 10 // 3 | let a = floor(10 / 3.0)
-
-*Help:* [expr5](https://vimhelp.org/eval.txt.html#expr5)
-
-------------------------------------------------------------------------------
-
-## Assignment Operators
-
-What|Python|VimScript
-----|------|---------
```

Comment:
Whoa.  I did not make these changes.  GitHub and/or browser text box autofill messed things up?:/

Reply:
Fixed it all up in a terminal with proper git.